samples/video/capture_and_display/README.rst

```diff
@@ -0,0 +1,60 @@
+.. _video_capture_display-sample:
+
+Video Capture And Display
+#########################
+
+Description
+***********
+
+This sample application uses the Video API to retrieve video frames from the
+video capture device, and writes frames to a video display device.
+
+Requirements
+************
+
+This sample requires a video capture device (e.g. a camera).
+
+- :ref:`mimxrt1064_evk`
+- `MT9M114 camera module`_
+- `RK043FN02H-CT TFT display`_
+
+Wiring
+******
+
+On :ref:`mimxrt1064_evk`, The MT9M114 camera module should be plugged in the
+J35 camera connector. A USB cable should be connected from a host to the micro
+USB debug connector (J41) in order to get console output via the freelink
+interface.
+
+Building and Running
+********************
+
+For :ref:`mimxrt1064_evk`, build this sample application with the following commands:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/video/capture_and_display
+   :board: mimxrt1064_evk
+   :goals: build
+   :compact:
+
+Sample Output
+=============
+
+You should see display printing frames.
+
+.. code-block:: console
+
+    - Display (ELCDIF_1) Capabilities:
+      RGBP width [480; 960; 1] height [272; 544; 1]
+      Default format: RGBP 480x272
+    - Camera (CSI) Capabilities:
+      RGBP width [640; 640; 0] height [480; 480; 0]
+      Default format: RGBP 640x480
+    Capture started
+
```

Comment:
Is this output on the console I presume? And does the camera capture will show on the display?

Reply:
Yes added description of display output